src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -156,21 +157,28 @@ public static boolean shouldBeHearingsInWelshFlag() {
 
     public static List<HearingLocations> getHearingLocations(CaseManagementLocation caseManagementLocation) {
         HearingLocations hearingLocations = new HearingLocations();
-        hearingLocations.setLocationId(caseManagementLocation.getBaseLocation());
-        hearingLocations.setLocationType("court");
+        List<HearingLocations> location;
+        List<String> multipleLocationList = new ArrayList<>(List.of("Manchester", "Chester", "Plymouth"));
```

Comment:
I would extract this into a json map keyed by the region so something like:

`"Manchester": {
["512401", "701411"
}.
"Chester": {
...`

That way you can load the file directly into a Map<String, List<String>> locationMap then do something like hearingLocations.addAll(locationMap.get(sscsCaseData.getProcessingVenue)

Reply:
Note: sscsCaseData change is in the linked PR